src/stac_iitmandi/models.py

```diff
@@ -20,8 +18,8 @@ def save(self):
         super().save()
 
         img = Image.open(self.image.path)
-        if img.height > 300 or img.width > 300:
-            output_size = (300, 300)
+        if img.height > 240 or img.width > 240:
+            output_size = (240, 240)
```

Comment:
The height and width can be stored in a variable in the file (e.g. `IMG_HEIGHT, IMG_WIDTH`) so when we want to change it will be easier. (Here and at line 43-44)

Reply:
@gajrajgchouhan , Global  variables are not recommended to be used in web dev especiall for django.

Reply:
![image](https://user-images.githubusercontent.com/56690827/102189346-8335cb00-3edc-11eb-946a-8edebde2072e.png)
![image (1)](https://user-images.githubusercontent.com/56690827/102190081-8b423a80-3edd-11eb-97e2-4e1484868d77.jpg)